packages/swapper/src/swappers/JupiterSwapper/errors.ts

```diff
@@ -0,0 +1,23 @@
+import type { SwapperErrorMapping } from '../baseErrors'
+import { SWAPPER_ERRORS } from '../baseErrors'
+
+export const JUPITER_ERROR_VALUES = {
+  SLIPPAGE_EXCEEDED: 'SlippageToleranceExceeded',
```

Comment:
```suggestion
  SLIPPAGE_TOLERANCE_EXCEEDED: 'SlippageToleranceExceeded',
```

Reply:
Changed in [2de18a2](https://github.com/shapeshift/web/pull/8433/commits/2de18a2c00a8ee7874b60ba277000ad720443b90)